### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by slhenry**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -53,13 +57,13 @@ This page introduces guidelines and other standards related to web accessibility
...
 
-All W3C standards are reviewed for accessibility support by the Accessible Platform Architectures ([APA](/about/groups/apawg/)) Working Group.
+All W3C standards are reviewed for accessibility support by the Accessible Platform Architectures ([APA](https://www.w3.org/WAI/about/groups/apawg/)) Working Group.
 
 The W3C standards and Working Group Notes introduced below are particularly relevant to accessibility.
 
 ## Accessibility Guidelines {#guidelines}
 
-[[Essential Components of Web Accessibility]](/fundamentals/components/) shows how web accessibility depends on several components of web development and interaction working together, and how the WAI guidelines (WCAG, ATAG, UAAG) apply.
+[Essential Components of Web Accessibility](https://www.w3.org/WAI/fundamentals/components/) shows how web accessibility depends on several components of web development and interaction working together, and how the WAI guidelines (WCAG, ATAG, UAAG) apply.
```
Comment (yatil) [inline index.md]: This change undoes the “take the title of the target translation on translated pages” that I implemented two weeks ago, including the automatic addition of “… (in English)” that the markdown links with double brackets do. Was that change intentional?

File: index.md [hunk 2]
```diff
@@ -68,11 +72,11 @@ Web "content" generally refers to the information in a web page or web applicati
 * natural information such as text, images, and sounds
 * code or markup that defines structure, presentation, etc.
 
-WCAG applies to dynamic content, multimedia, "mobile", etc. WCAG can also be applied to non-web information and communications technologies (ICT), as described in [WCAG2ICT](/standards-guidelines/wcag/non-web-ict/).
+WCAG applies to dynamic content, multimedia, "mobile", etc. WCAG can also be applied to non-web information and communications technologies (ICT), as described in [WCAG2ICT](https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/).
```
Comment (yatil) [inline index.md]: same

File: index.md [hunk 3]
```diff
@@ -68,11 +72,11 @@ Web "content" generally refers to the information in a web page or web applicati
...
 
 WCAG info:
-- [WCAG Overview](/standards-guidelines/wcag/)
-- [[WCAG 2.1 at a Glance]](/standards-guidelines/wcag/glance/)
+- [WCAG Overview](https://www.w3.org/WAI/standards-guidelines/wcag/)
+- [WCAG 2.1 at a Glance]( https://www.w3.org/WAI/standards-guidelines/wcag/glance/)
```
Comment (yatil) [inline index.md]: same

File: index.md [hunk 4]
```diff
@@ -84,16 +88,16 @@ Authoring tools are software and services that "authors" (web developers, design
 * help authors create more accessible web content.
 
 ATAG info:
-- [ATAG Overview](/standards-guidelines/atag/)
-- [[ATAG at a Glance]](/standards-guidelines/atag/glance/)
+- [ATAG Overview](https://www.w3.org/WAI/standards-guidelines/atag/)
+- [ATAG at a Glance](https://www.w3.org/WAI/standards-guidelines/atag/glance/)
```
Comment (yatil) [inline index.md]: same

File: index.md [hunk 5]
```diff
@@ -84,16 +88,16 @@ Authoring tools are software and services that "authors" (web developers, design
...
 - [ATAG 2.0 Standard](https://www.w3.org/TR/ATAG/)
 
 ### User Agent Accessibility Guidelines (UAAG) {#uaag}
 
 User agents include browsers, browser extensions, media players, readers, and other applications that render web content.
 
 UAAG info:
-- [UAAG Overview](/standards-guidelines/uaag/)
+- [UAAG Overview](https://www.w3.org/WAI/standards-guidelines/uaag/)
```
Comment (yatil) [inline index.md]: same (but should be `[[UAAG Overview]](/standards-guidelines/uaag/)`)

File: index.md [hunk 6]
```diff
@@ -105,7 +109,7 @@ ARIA provides semantics so authors can convey user interface behaviors and struc
 The ARIA suite includes <abbr title="application programming interface">API</abbr> mapping specifications that provide user agent implementation guidance. It also include modules for Graphics and Digital Publishing.
 
 ARIA info:
-- [WAI-ARIA Overview](/standards-guidelines/aria/) – includes a [list and description of modules and API mappings](/standards-guidelines/aria/#versions)
+- [WAI-ARIA Overview](https://www.w3.org/WAI/standards-guidelines/aria/) – includes a [list and description of modules and API mappings](https://www.w3.org/WAI/standards-guidelines/aria/#versions)
```
Comment (yatil) [inline index.md]: same

File: index.md [hunk 7]
```diff
@@ -24,6 +28,7 @@ footer: >   # Translate all the words below, including "Date:" and "Editor:". Do
 # end of translation instructions
 ---
 
+@@@@@@@  +  new icons for links-to-w3c-page-without-wai-nav @@@@@@@
```
Comment (yatil) [inline index.md]: The new icon is used automatically when `{% include different.html %}` is used. (Can have label attribute, like {% include different.html label="Looks differently" %}).

Comment (slhenry) [reply index.md]: Please document this where it is easy for editors to find. Thanks!

Comment (yatil) [reply index.md]: Happy to do so when writing documentation is a priority. It has explicitly not been one before.
